src/zrc/api/tests/test_audittrails.py

```diff
@@ -159,10 +160,12 @@ def test_partial_update_zaak_audittrails(self):
     def test_create_zaakinformatieobject_audittrail(self):
         zaak_data = self._create_zaak()
 
-        zaak_uuid = zaak_data['url'].split('/')[-1]
-        url = reverse(ZaakInformatieObject, kwargs={'zaak_uuid': zaak_uuid})
+        url = reverse('zaakinformatieobject-list', kwargs={
```

Comment:
if you use the `vng_api_common.test.reverse` function, you can use `reverse(ZaakInformatieObject)` to get the same result. It looks at the default version and does some magic to figure out the list or detail action.